server.py

```diff
@@ -56,39 +66,88 @@ async def upload(file: UploadFile=File(...)):
 
     return {"message": f"{file.filename} received"}
 
-@app.post('/api/csv')
+# simple pipeline query with no side effects
+# TESTED, WORKING 7/29/24
+# required body for Postman
+    # body -> raw -> json
+    # {
+    #     "query": "how tall are banana trees"
+    # }
+@app.post('/api/query')
```

Comment:
want to know more about this route after we get the feature validation screenshots!